numpy/linalg/tests/test_linalg.py

```diff
@@ -61,30 +61,44 @@ def get_rtol(dtype):
 
 class LinalgCase(object):
 
-    def __init__(self, name, a, b, exception_cls=None):
+    def __init__(self, name, a, b, flags=frozenset()):
```

Comment:
Why not a simple list instead of a `frozenset`? I don't see that an immutable set really matters here.